CHANGES.md

```diff
@@ -12,7 +12,25 @@
 -   Support for appending to an existing data source when supported by GDAL by
     passing `append=True` to `pyogrio.write_dataframe` (#197).
 
-## O.4.2
+### Improvements
+
+-   It is now possible to pass GDAL's dataset creation options in addition
+    to layer creation options in `pyogrio.write_dataframe` (#189).
+-   When specifying a subset of `columns` to read, unnecessary IO or parsing
+    is now avoided (#195).
+-   In floating point columns, NaN values are now by default written as "null"
```

Comment:
Should we list this as a potentially breaking change?

Reply:
As a separate section? Or just mention that is is potentially breaking in this list item? 
(I am just not sure how much this will affect people in practice, although I just checked and for example with opening a shapefile in QGIS, is does differ, i.e. it now sees it as missing value instead of as NaN)

Reply:
As a separate section, as a sibling of enhancements:

```
### Potentially breaking changes

-  In floating point columns, NaN values are now...

```